test/test-v8debugapi.js

```diff
@@ -1171,6 +1171,32 @@ describe('v8debugapi', function() {
         process.nextTick(function() {foo(1);});
       });
     });
+
+    it('should capture state in transpiled TS async functions', (done) => {
+      const bp = {
+        id: 'async-id-1',
+        location: {
+          path: path.join('.', 'test', 'fixtures', 'ts', 'async.js'),
+          line: 71
+        }
+      };
+
+      const run = require('./fixtures/ts/async.js');
+      api.set(bp, (err) => {
+        assert.ifError(err);
+        process.nextTick(run);
```

Comment:
The point about consistency is a good one. I do think this way is more desirable as the other way has an async race. I will make it consistent with the other tests for now though.